command/helper/helper.go

```diff
@@ -414,6 +414,7 @@ func ReadConfig(baseCommand string, args []string) (*Config, error) {
 	flags.StringVar(&cliConfig.JSONRPCAddr, "jsonrpc", "", "")
 	flags.StringVar(&cliConfig.Join, "join", "", "")
 	flags.StringVar(&cliConfig.Network.Addr, "libp2p", "", "")
+	flags.StringVar(&cliConfig.METRICSAddr, "metrics", "", "")
```

Comment:
This should be called `prometheus`.

Reply:
Done,Here is the commit [link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/ce11ec1c9f644ed16644db8da3718955cbf82e86).